webapp/app/config.py

```diff
@@ -81,6 +81,15 @@ class DevelopmentConfig(BaseConfig):
     REACT_BUNDLE_NAME = 'bundle.js'
 
 
+class AcceptanceConfig(DevelopmentConfig):
+    DEBUG = False
+    DEBUG_DATA = False
+    USE_MOCK_API = True
```

Comment:
Yeah, sorry the naming of "Acceptance" for the env is confusing. I'll rethink that. To pick up your other question:

> I'm not sure I get the setup right between the acceptance_tests/cypress/integration/lotse_flow.spec.js and the client/cypress/integration/login.spec.js. I feel like they're more independent than I understood so far. Maybe you could explain when which of these tests is executed?

They are completely separate, not connected in any way, shape or form. They just both happen to be implemented with Cypress.

**Acceptance Tests**
The acceptance suite is our "but does it work in real life" final suite of checks that does - as you say - a complete runthrough including erica. We should not have a lot of tests here, just test user flows what may never break, such as logging in, submitting a tax return, and being presented with the correct wording on compliance checkboxes.

**End-to-End webapp tests**
These are for testing the integration of frontend components with the flask backend and for testing the webapp _as a user would use it_ through a real browser. We can have a lot more tests here, but because they run in a real browser, they are always going to be slow. Something that's currently in acceptance tests but belongs here IMO is e.g. exhaustive testing of redirects in the tax form.

Does that help?

Reply:
Yes, that does help, thank you. Especially the explicit naming of "End-to-End **webapp** tests" - For me, end to end is from the user to Elster and back^^